aiven/client/argx.py

```diff
@@ -146,7 +146,7 @@ def save(self) -> None:
 class CommandLineTool:  # pylint: disable=old-style-class
     def __init__(self, name: str, parser: Optional[argparse.ArgumentParser] = None):
         self.log = logging.getLogger(name)
-        self.config: Optional[Config] = None
+        self.config: Config = cast(Config, {})
```

Comment:
it's to please `mypy` that this config is a `Config` object. Although the config is actually constructed in `CommandLine.run`, `mypy` doesn't know about it, so we need to tell that this config won't be `None` and it will be `Config` object.

Reply:
maybe it would work the same way as you suggested for the `client: AivenClient` :+1:  I'll try that